src/editable-heading/editable-heading.tsx

```diff
@@ -51,7 +51,7 @@ export const EditableHeading = (props: EditableHeadingProps) => {
     autoFocus = true, 'data-test': dataTest, error, disabled, multiline = false,
     renderMenu = () => null,
     onFocus, onBlur, onChange,
-    onScroll, maxInputHeight,
+    onScroll, maxInputRows,
```

Comment:
Don't we need to have some default here?

Reply:
If I remember correctly, we decided to make the textarea have no constraints in case lh doesn't work, but I think the same behavior should also work for if we passed nothing in

Reply:
I agree on that. But is it working now? Looks like, if we pass nothing, it would be `max-height: undefinedlh;` which looks weird even if works fine

Reply:
fixed that

Reply:
Thanks!